app/views/order_management/order_details/_print_details.html.haml

```diff
@@ -0,0 +1,38 @@
+.show-for-print
+  .banner-list
+    .row
+      %dl
+        %dt= @order_detail.class.human_attribute_name(:account)
+        %dd= @order_detail.account
+    - if @order_detail.project
```

Comment:
We should probably use a view hook for this. In the forks that don't have the projects engine enabled, `OrderDetail` won't respond to `project`.